src/components/DataInput/DatePicker.tsx

```diff
@@ -0,0 +1,13 @@
+import styled from "styled-components";
+import { DatePicker } from "@mui/x-date-pickers/DatePicker";
+
+const StyledDatePicker = styled(DatePicker)`
+    & .MuiInputBase-root {
+        & .MuiInputBase-input {
+            // pushes calendar icon to the right
+            width: 10000%;
+        }
+    }
+`;
```

Comment:
This might be tidier?

```
const StyledDatePicker = styled(DatePicker)`
    & .MuiInputBase-root {
        & .MuiInputBase-input {
            // pushes calendar icon to the right
            flex-grow: 1;
            flex-shrink: 1;
        }
    }
    & .MuiInputAdornment-root {
        width: 40px;
    }
`;
```

Reply:
nvm I misunderstood this comment, will try this!

Reply:
So yeah just to confirm, targeting like suggested didn't work, I would have thought it would too tbh.